packages/harmony/src/components/layout/Popup/Popup.tsx

```diff
@@ -0,0 +1,411 @@
+          originalTopPosition.current - scrollTop + initialScrollPosition
+        }px`
+      }
+    },
+    [wrapperRef, originalTopPosition]
+  )
+
+  // Set up scroll listeners
+  useEffect(() => {
+    if (isVisible && anchorRef.current) {
+      const scrollParent = getScrollParent(anchorRef.current)
+      if (!scrollParent) return
+
+      const initialScrollPosition = scrollParent.scrollTop
+      const listener = () => watchScroll(scrollParent, initialScrollPosition)
+      scrollParent.addEventListener('scroll', listener)
+      return () => {
+        scrollParent.removeEventListener('scroll', listener)
+      }
+    }
+
+    return () => {}
+  }, [isVisible, watchScroll, anchorRef])
+
+  // Set up key listeners
+  useEffect(() => {
+    if (isVisible) {
+      const escapeListener = (e: KeyboardEvent) => {
+        if (e.code === 'Escape') {
+          handleClose()
+        }
+      }
+
+      window.addEventListener('keydown', escapeListener)
+
+      return () => window.removeEventListener('keydown', escapeListener)
+    }
+    return () => {}
+  }, [isVisible, handleClose])
+
+  const transitions = useTransition(isVisible, null, {
+    from: {
+      transform: `scale(0)`,
+      opacity: 0
+    },
+    enter: {
+      transform: `scale(1)`,
+      opacity: 1
+    },
+    leave: {
+      transform: `scale(0)`,
+      opacity: 0
+    },
+    config: theme.spring.standard,
+    unique: true
+  })
+
+  const rootStyle = zIndex ? { zIndex } : {}
+
+  const handleMouseLeave = useCallback(() => {
+    if (dismissOnMouseLeave) {
+      onClose?.()
+    }
+  }, [dismissOnMouseLeave, onClose])
+
+  return (
+    <>
+      {/* Portal the popup out of the dom structure so that it has a separate stacking context */}
+      {ReactDOM.createPortal(
+        <div
+          ref={wrapperRef}
+          className={styles.root}
+          style={rootStyle}
+          onMouseLeave={handleMouseLeave}
+        >
+          {transitions.map(({ item, key, props }) =>
+            item ? (
+              <animated.div
+                className={styles.popup}
+                ref={popupRef}
+                key={key}
+                style={{
+                  ...props,
+                  transformOrigin: `${computedTransformOrigin.horizontal} ${computedTransformOrigin.vertical}`
+                }}
+              >
+                {showHeader && (
+                  <div
+                    className={cn(styles.header, {
+                      [styles.noAfter]: hideCloseButton
+                    })}
+                  >
+                    {hideCloseButton ? null : (
+                      <PlainButton
+                        variant={PlainButtonType.SUBDUED}
+                        aria-label={messages.close}
+                        onClick={handleClose}
+                        iconLeft={IconClose}
+                      />
+                    )}
+                    <div className={styles.title}>{title}</div>
```

Comment:
should use `Text` component for the text here and then can delete the font and color styles from `styles.title` class